2017/03/02 18:27:29 [error] 15893#15893: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 0.0.0.0, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:23:46 [error] 16887#16887: *8 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:27:21 [error] 16887#16887: *13 open() "/home/xuchu/xcpy/xcDjango/acfun_qa_home/static_all/static/local_css.css" failed (2: No such file or directory), client: 192.168.35.17, server: 127.0.0.1, request: "GET /static/local_css.css HTTP/1.1", host: "192.168.35.9:8001"
2017/03/02 19:27:36 [error] 16887#16887: *13 open() "/home/xuchu/xcpy/xcDjango/acfun_qa_home/static_all/static/local_css.css" failed (2: No such file or directory), client: 192.168.35.17, server: 127.0.0.1, request: "GET /static/local_css.css HTTP/1.1", host: "192.168.35.9:8001"
2017/03/02 19:30:36 [error] 17029#17029: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:33:12 [error] 17100#17100: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:33:39 [error] 17111#17111: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:33:40 [error] 17111#17111: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:33:41 [error] 17111#17111: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:33:41 [error] 17111#17111: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:51:26 [error] 17365#17365: *44 upstream prematurely closed connection while reading response header from upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/02 19:53:09 [error] 17492#17492: *46 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8020/", host: "192.168.35.9:8001"
2017/03/02 19:54:27 [error] 17505#17505: *49 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://192.168.35.9:8020/", host: "192.168.35.9:8001"
2017/03/02 19:56:07 [error] 17538#17538: *51 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://192.168.35.9:8020/", host: "192.168.35.9:8001"
2017/03/02 19:56:09 [error] 17538#17538: *51 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://192.168.35.9:8020/", host: "192.168.35.9:8001"
2017/03/02 19:56:20 [error] 17538#17538: *51 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://192.168.35.9:8020/", host: "192.168.35.9:8001"
2017/03/02 19:56:29 [error] 17538#17538: *51 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://192.168.35.9:8020/", host: "192.168.35.9:8001"
2017/03/02 19:59:18 [error] 17598#17598: *65 upstream prematurely closed connection while reading response header from upstream, client: 192.168.35.17, server: 127.0.0.1, request: "GET /admin/login/?next=/admin/ HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/03 09:33:18 [error] 1189#1189: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.24, server: 127.0.0.1, request: "GET /admin/login/?next=/admin/ HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001"
2017/03/03 09:33:21 [error] 1189#1189: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.35.24, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://127.0.0.1:8020", host: "192.168.35.9:8001", referrer: "http://192.168.35.9:8001/admin/login/?next=/admin/"
